src/ios/Fingerprint.swift

```diff
@@ -23,13 +23,24 @@ import LocalAuthentication
     var reason = "Authentication";
     let data  = command.arguments[0] as AnyObject?;
 
-    if let clientId = data?["clientId"] as! String? {
-      reason = clientId;
-    }
+    var policy:LAPolicy = .deviceOwnerAuthenticationWithBiometrics;
+    if let disableBackup = data?["disableBackup"] as! Bool? {
+        if disableBackup {
+            authenticationContext.localizedFallbackTitle = "";
+            policy = .deviceOwnerAuthenticationWithBiometrics;
+        }else{
+          if let localizedFallbackTitle = data?["localizedFallbackTitle"] as! String? {
+            authenticationContext.localizedFallbackTitle = localizedFallbackTitle;
+          }
 
+          if #available(iOS 9.0, *) {
+              policy = .deviceOwnerAuthentication;
```

Comment:
Thanks for clarifying this. I was not aware of this behavior of the iOS API. Setting the policy should be more straightforward.